Resources/Prototypes/Recipes/Lathes/powercells.yml

```diff
@@ -32,6 +32,18 @@
     Plastic: 200
     Gold: 50
 
+- type: latheRecipe
+  id: PowerCellHyper
+  result: PowerCellHyperPrinted
+  categories:
+  - Parts
+  completetime: 10
+  materials:
+    Steel: 600
+    Glass: 600
+    Plasma: 300
+    Gold: 200
```

Comment:
i think this is a little bit too close to the high powercell for my taste? Microreactor uses uranium which is cool so maybe this should use silver or something just to separate it. Creeping up the costs like it's done here feels very minimal.